docs/tutorials-samples/targeting-core-for-new-libraries.md

```diff
@@ -0,0 +1,298 @@
+
+**NOTE:** This assumes your code will compile across *both* .NET Core and .NET Framework.  Read the section on cross-compiling with `#if`s on how to compile the same file differently for each target if you are using features which are unavailable in some of your targets.
+
+## How do I target a PCL?
+
+Targeting a PCL profile is a bit trickier.  For starters, [reference this list of PCL profiles](http://embed.plnkr.co/03ck2dCtnJogBKHJ9EjY/preview) to ensure you have the correct target.  Hover over the Name of each entry for the framework identifier, which you will need.
+
+You will then need to do two things:
+
+1. List the dependencies for each target *inside* of that target.  The "global" `dependencies` section cannot be used.
+2. List the framework assemblies you are using in your code *inside* the PCL target profile section.  This will likely require, at a minimum, `mscorlib`, `System`, and `System.Core`.
+
+Here is an example targeting PCL Profile 328 and .NET Core, using only `System.Runtime` as a dependency.
+
+```
+{
+    "frameworks":{
+        "dotnet51":{
+           "dependencies":{
+                "System.Runtime":"4.0.0-rc1-*"
+            }
+        },
+        "dotnet55":{
+           "dependencies":{
+                "System.Runtime":"4.0.0-rc1-*"
+            }
+        },
+        ".NETPortable,Version=v4.0,Profile=Profile328":{
+            "frameworkAssemblies":{
+                "mscorlib":"",
+                "System":"",
+                "System.Core":""
+            }
+        }
+    }
+}
+```
+
+This will allow you to target .NET Core 5.1, .NET Core 5.5, .NET Framework 4.0, Windows 8, Windows Phone 8.1, WIndows Phone Silverlight 8.1, and Silverlight 5.0.
+
+Run `dnu restore` and `dnu build` from the command line, and your library can now be built!  You will now notice three new entries in your `/bin/Debug` folder:
+
+```
+/bin
+   /Debug
+      /dotnet51
+      /dotnet55
+      /portable-net40+sl50+netcore45+wpa81+wp8
+```
+
+Finally, run `dnu pack` to build a NuGet pakage, and your `/bin/Debug` folder should look like this:
+
+```
+/bin
+   /Debug
+      /dotnet51
+      /dotnet55
+      /portable-net40+sl50+netcore45+wpa81+wp8
+      Lib.1.0.0.nupkg
+      Lib.1.0.0.symbols.nupkg
+```
+
+And now you can publish a NuGet package!
+
+**NOTE:** This assumes your code will compile across *both* .NET Core and .NET Framework.  Read the section on cross-compiling with `#if`s on how to compile the same file differently for each target if you are using features which are unavailable in some of your targets.
+
+## How do I cross-compile for .NET Core and .NET Framework?
+
+If you need to support .NET Framework and .NET Core, you may find yourself needing to use different APIs to accomplish the same task.  You may also wish to support different language constructs (such as `async` and `await`) and support modern APIs within the same file.  A great way to do this is to cross-compile with `#if` guards.
```

Comment:
"such as async and await" could be "such as async and await, which only became available with .NET x.x" to better get the point across.